libutil/dr_config.c

```diff
@@ -473,7 +473,9 @@ get_config_dir(bool global, char *fname, size_t fname_len, bool find_temp)
     /* On failure, we still want to copy the last-tried dir out so drdeploy can have a
      * nicer error msg.
      */
+    #pragma GCC diagnostic ignored "-Wformat-truncation"
```

Comment:
Will VS give an error on an unknown pragma?  Ditto below.

Does s/fname_len/fname_len - 1/ not remove the warning?

Reply:
It does not:

`‘/’ directive output may be truncated writing 1 byte into a region of size between 0 and 511 [-Werror=format-truncation=]`

I'll try and see if I can test this under VS to figure out if this is an issue.

Reply:
I've "solved" this by first checking if we are in GCC, so other compilers will ignore.

Done?